app/_how-tos/generate-certificate-with-sni.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Associate a Certificate with an SNI
+content_type: how_to
+related_resources:
+  - text: SNI entity
+    url: /gateway/entities/snis/
+  - text: Certificates
+    url: /gateway/entities/certificate/
+
+
+products:
+    - gateway
+tier: enterprise
+
+works_on:
+    - on-prem
+
+entities: 
+  - certificate
+  - sni
+  - route
+  - service
+
+tldr:
+    q: How do you validate that a Certificate is attached to an SNI and working?
+    a: After creating the Certificate and associating it to the desired SNI, then using `openssl` you can verify that {{site.base_gateway}} is returning the expected certificate for the SNI. 
+tools:
+    - deck
+
+prereqs:
+  inline:
+    - title: Certificate
+      include_content: prereqs/certificate
+      icon_url: /assets/icons/file.svg
+    - title: Configure environment variables
+      content: |
+        Set the following variables: 
+        * `CERT_ID`: The UUID set when you associated a Certificate with {{site.base_gateway}}.
+        * `SNI_NAME`: The name of the [SNI](/gateway/entities/sni/)
```

Comment:
This is missing some context. Do I need to have an SNI entity already created? Do I just need an SNI itself?